spec/springboard/client/resource_spec.rb

```diff
@@ -48,6 +48,12 @@
         it "should add bracket notation for array parameters" do
           expect(resource.__send__(method, :somearray => [1, 2, 3]).uri.to_s).to eq("/some/path?somearray[]=1&somearray[]=2&somearray[]=3")
         end
+
+        it "should return a new resource without modifying the existing URI" do
+          new_resource = resource.query(per_page: 1)
```

Comment:
Need to use `__send__(method, ...)` here like the other specs.

Reply:
Fixed